BioMed Central.js

```diff
@@ -0,0 +1,258 @@
+{
+	"translatorID": "75ccea3c-fdb8-4473-8203-ceb27f3395f8",
+	"label": "BioMed Central",
+	"creator": "Philipp Zumstein",
+	"target": "^https?:\\/\\/[^\\.]+\\.biomedcentral\\.com\\/(articles|search)",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2015-12-29 21:36:02"
+}
+
+/*
+    ***** BEGIN LICENSE BLOCK *****
+
+    Copyright © 2015 Philipp Zumstein
+
+    This file is part of Zotero.
+
+    Zotero is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Zotero is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+    ***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+    if (url.indexOf('biomedcentral.com/articles/10.1186/')>-1) {
+        return "journalArticle";
+    } else if (getSearchResults(doc, true)) {
+        return "multiple";
+    }
+}
+
+function getSearchResults(doc, checkOnly) {
+    var items = {};
+    var found = false;
+    var rows = ZU.xpath(doc, '//div[@id="search-container"]//article//h3[contains(@class,"ResultsList_title")]/a');
+    for (var i=0; i<rows.length; i++) {
+        var href = rows[i].href;
+        var title = ZU.trimInternal(rows[i].textContent);
+        if (!href || !title) continue;
+        if (checkOnly) return true;
+        found = true;
+        items[href] = title;
+    }
+    return found ? items : false;
+}
+
+
+function doWeb(doc, url) {
+    if (detectWeb(doc, url) == "multiple") {
+        Zotero.selectItems(getSearchResults(doc, false), function (items) {
+            if (!items) {
+                return true;
+            }
+            var articles = new Array();
+            for (var i in items) {
+                articles.push(i);
+            }
+            ZU.processDocuments(articles, scrape);
+        });
+    } else {
+        scrape(doc, url);
+    }
+}
+
+function scrape(doc, url) {
+	var DOI = url.match(/\/(10\.[^#?]+)/)[1];
+	var risURL = "http://citation-needed.services.springer.com/v2/references/" + DOI + "?format=refman&flavour=citation";
```

Comment:
well -- it's a parameter, that's why I thought `encodeURIComponent`, but it breaks because the slash is treated like a regular slash in the URL, not a parametere element. I'd just leave it alone then. I think we can assume that BMC doesn't use  `, ? : @ & = + $ #` in their DOIs. (and using encodeURI wouldn't help for those).